File: Euler_5/Euler_5.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start, finish, lcm := 1, 20, 1
	sum := lcm

	for i := start; i <= finish; i++ {
		for sum%i != 0 {
			sum += lcm
		}
		lcm = sum
	}
	fmt.Println(sum)
	defer timeTrack(time.Now(), "Euler_6")
}

func timeTrack(start time.Time, name string) {
	elapsed := time.Since(start)
	fmt.Printf("%s took %s\n", name, elapsed)
}

File: Euler_005/Euler_5.java
class Euler_5{
  public static void main(String[] args) {
    int start = 1;
    int finish = 20;
    int lcm = 1;
    int sum = lcm;

    for(int i = start; i < finish; i++){
      while (sum % i != 0){
        sum += lcm;
      }
      lcm = sum;
    }
    System.out.println(sum);
  }
}
